README.md

```diff
@@ -1,4 +1,5 @@
 # Project 2: Battle Game
+<!-- Interesting! -->
```

Comment:
You should not change the code here, which I guess is just a test at first.

Reply:
Yes it's a test and I have changed it back.